vendors/pc/boards/windows/ports/pkcs11/iot_pkcs11_pal.c

```diff
@@ -156,7 +159,7 @@ CK_OBJECT_HANDLE PKCS11_PAL_SaveObject( CK_ATTRIBUTE_PTR pxLabel,
         /* Create the file. */
         hFile = CreateFileA( pcFileName,
                              GENERIC_WRITE,
-                             0,
+                             FILE_SHARE_READ,
```

Comment:
This doesn't make sense to me. We're opening the file to blow it away and rewrite here. Sharing it for read won't result in anything good. If two threads can be calling this routine at the same time, it's a bug.

Reply:
Yes good catch I will change that back to zero.